library/src/interactor_impl.cxx

```diff
@@ -323,6 +323,10 @@ class interactor_impl::internals
         self->SetViewOrbit(ViewType::VT_FRONT, self);
         render = true;
         break;
+      case '2':
```

Comment:
lets use 4 and 6 instead :)

Reply:
Okay but please confirm, 4 for -90 degree rotation and 6 for +90 degree rotation?

Reply:
yes